tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>RK-Cards</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+  </head>
+  <body>
+    <div class="main">
+      <div class="row">
+        <div class="row-1">
+          <img
+            alt="Image"
+            height="100px"
+            width="100px"
+            src="https://cdn-icons-png.flaticon.com/512/681/681662.png"
+          />
+          <h3>Design</h3>
+        </div>
+        <div class="row-2">
+          <p>
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum,
+            aliquam eum quis corrupti placeat at modi iure temporibus assumenda
+            quo, recusandae ab similique deserunt. At magnam esse ipsum neque
+            quasi.
+          </p>
+          <a>Read More</a>
+        </div>
+      </div>
+      <div class="row">
+        <div class="row-1">
+          <img
+            alt="Image"
```

Comment:
alt must describe about image if it is not rendered! 
what does `alt="Image"` describe ?

Reply:
Done @Abbas4tech